### PR TITLE
Add support for Distribution metric in go-metrics

### PR DESCRIPTION
**Pull request by SubChord**

### CODE REVIEW

**Comment by deathowl** [comment]
hey @SubChord  could you please fix the build, would love to merge this

**Comment by SubChord** [comment]
> hey @SubChord could you please fix the build, would love to merge this

At first glance I can not tell why the commit breaks that test. I never touched the Gauged Meter part...

**Comment by deathowl** [comment]
https://github.com/deathowl/go-metrics-prometheus/pull/15/commits/c3e343906de25cf497b1fe013da17e1fac98a14e this causes the build faillure

**Comment by deathowl** [comment]
Please rebase onto master to resolve conflicts, and make build pass, else i have nothing against merging it.

**Comment by SubChord** [comment]
Done :)

File: go.mod [hunk 1]
```diff
@@ -1,8 +1,8 @@
-module github.com/deathowl/go-metrics-prometheus
+module github.com/subchord/go-metrics-prometheus
```
Comment (huydang284) [inline go.mod:1]: Hi @SubChord, please revert it. Thanks.

File: go.mod [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
 
 go 1.13
 
 require (
-	github.com/prometheus/client_golang v1.1.0
-	github.com/rcrowley/go-metrics v0.0.0-20190826022208-cac0b30c2563
+	github.com/prometheus/client_golang v1.2.1
+	github.com/subchord/go-metrics v1.0.1
```
Comment (huydang284) [inline go.mod:7]: Hi @SubChord, we should specify the version, but I think we should keep the original repository `github.com/rcrowley/go-metrics`

Comment (SubChord) [reply go.mod:7]: I agree but the rcrowley/go-metrics hasn't been updated in months. It does not support the distribution metric.

Comment (deathowl) [reply go.mod:7]: Sorry I dont want to depend on a Fork. Please get rcrowley to merge your pr, and then I can merge yours.

**Pull request closed without merging**